apps/frontend/src/modules/users/slices/actions.ts

```diff
@@ -0,0 +1,25 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+
+import { type AsyncThunkConfig } from "~/libs/types/types.js";
+import { type UserDto, type UserUpdatePayload } from "~/modules/users/users.js";
+
+import { name as sliceName } from "./users.slice.js";
+
+const getUserFromAuth = createAsyncThunk<UserDto, undefined, AsyncThunkConfig>(
```

Comment:
Why do we need this as an action?

Reply:
I have to get user from auth store and put it into users store. What a correct way to do it?

Reply:
Use a user from auth slice directly in the component, no need to move it to user slice

Reply:
then how should I update user in users slice, not moving user to this slice

Reply:
Let's create a new endpoint getById and get the user from there. Don't forget to add a preHandler check